src/services/refactors/inlineLocal.ts

```diff
@@ -0,0 +1,173 @@
+/* @internal */
+namespace ts.refactor.inlineLocal {
+    const refactorName = "Inline local";
+    const refactorDescription = getLocaleSpecificMessage(Diagnostics.Inline_local);
+
+    const inlineHereActionName = "Inline here";
+    const inlineAllActionName = "Inline all";
+
+    const inlineHereActionDescription = getLocaleSpecificMessage(Diagnostics.Inline_here);
+    const inlineAllActionDescription = getLocaleSpecificMessage(Diagnostics.Inline_all);
+
+
+    registerRefactor(refactorName, { getEditsForAction, getAvailableActions });
+
+    interface Info {
+        declaration: VariableDeclaration;
+        usages: ReadonlyArray<Identifier>;
+        selectedUsage: Identifier | undefined;
+    }
+
+    function getAvailableActions(context: RefactorContext): ApplicableRefactorInfo[] | undefined {

```

Comment:
change to new interface `ReadonlyArray<ApplicableRefactorInfo>`  instead of `ApplicableRefactorInfo[] | undefined`